src/go/k8s/pkg/console/console.go

```diff
@@ -0,0 +1,27 @@
+package console
+
+import (
+	"github.com/cloudhut/common/rest"
+	"github.com/redpanda-data/console/backend/pkg/connect"
+	"github.com/redpanda-data/console/backend/pkg/kafka"
+)
+
+const (
+	debugLogLevel = 4
+)
+
+// ConsoleConfig is the config passed to the Redpanda Console app
+type ConsoleConfig struct {
```

Comment:
isn't there version of this config inside console so we can just import it as we do for redpanda via rpk?

Reply:
As discussed, I've added a comment about it - the config types in Console don't have Enterprise fields and we don't support all fields yet